tsconfig-for-webpack.json

```diff
@@ -1,7 +1,25 @@
 {
+  "ts-node": {
+    // This can be omitted when using ts-patch
+    "compiler": "ttypescript",
+    "transpileOnly": true,
+    "moduleTypes": {
+      "webpack/*": "cjs",
+      // Globs are also supported with the same behavior as tsconfig "include"
+      "webpack/**/*": "cjs"
+    }
+  },
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es5",
+    "plugins": [
+      {
+        "transform": "@zoltu/typescript-transformer-append-js-extension/output/index.js",
```

Comment:
Can this file use `extends` like `tsconfig.json` does, so it can share these settings with tsconfig.json and tsconfig.base.json?